### DISCUSSION TITLE
How to limit single-core operation？

### DISCUSSION THREAD

**Discussion by meteor-lx95** [autogluon/autogluon#3118] [Q&A] [answered]
How to limit single-core operation,I want to see the single core running time.(num_cpus=1)

**Comment by gradientsky** [comment, marked as answer]
Please use `num_cpus` parameter - see the [documentation](https://auto.gluon.ai/stable/tutorials/tabular/tabular-faq.html#how-to-limit-the-number-of-cores-autogluon-will-use). Another way to hard-cap number of CPUs is to run in docker with only specified number of cores.
